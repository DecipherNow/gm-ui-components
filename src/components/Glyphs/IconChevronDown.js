import React from "react";
import Icon from "../Icon";

export default function ChevronDown(props) {
  return (
    <Icon {...props} glyphName="ChevronDown">
      <path
        d="M13.4393398,12.5 L8.9696699,8.03033007 C8.67677668,7.73743685 8.67677668,7.26256311 8.9696699,6.96966989 C9.26256312,6.67677667 9.73743686,6.67677667 10.0303301,6.96966989 L15.5606602,12.5 L10.0303301,18.0303301 C9.73743686,18.3232233 9.26256312,18.3232233 8.9696699,18.0303301 C8.67677668,17.7374369 8.67677668,17.2625632 8.9696699,16.9696699 L13.4393398,12.5 Z"
        id="ChevronDown"
         
        fillRule="nonzero"
        transform="translate(12.155330, 12.500000) scale(-1, 1) rotate(90.000000) translate(-12.155330, -12.500000) "
      />
    </Icon>
  );
}

ChevronDown.displayName = "ChevronDown";
