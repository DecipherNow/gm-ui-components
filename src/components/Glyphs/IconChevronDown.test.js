import React from "react";
import { shallow } from "enzyme";
import IconChevronDown from "./IconChevronDown.svg";
describe("IconChevronDown", () => {
  it("matches snapshot", () => {
    
    const aIconChevronDown = shallow(<IconChevronDown />).find("IconChevronDown.svg").dive();
    console.log(aIconChevronDown.debug())
    expect(aIconChevronDown).toMatchSnapshot();
  });
});