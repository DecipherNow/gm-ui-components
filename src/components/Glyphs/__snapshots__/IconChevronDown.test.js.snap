// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconChevronDown matches snapshot 1`] = `
<Icon
  backgroundColor="currentColor"
  backgroundOpacity={1}
  backgroundSizeRatio={1}
  borderColor="currentColor"
  borderOpacity={1}
  borderWidth={1}
  glyph=""
  glyphColor="currentColor"
  glyphName="ChevronDown"
  glyphSizeRatio={1}
>
  <path
    d="M13.4393398,12.5 L8.9696699,8.03033007 C8.67677668,7.73743685 8.67677668,7.26256311 8.9696699,6.96966989 C9.26256312,6.67677667 9.73743686,6.67677667 10.0303301,6.96966989 L15.5606602,12.5 L10.0303301,18.0303301 C9.73743686,18.3232233 9.26256312,18.3232233 8.9696699,18.0303301 C8.67677668,17.7374369 8.67677668,17.2625632 8.9696699,16.9696699 L13.4393398,12.5 Z"
    fillRule="nonzero"
    id="ChevronDown"
    transform="translate(12.155330, 12.500000) scale(-1, 1) rotate(90.000000) translate(-12.155330, -12.500000) "
  />
</Icon>
`;
